.github/workflows/build.yml

```diff
@@ -0,0 +1,30 @@
+name: Build
+
+on:
+  pull_request:
+    branches: [master]
+
+jobs:
+  build:
+    name: Build shared libraries
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+    steps:
+      - uses: actions/checkout@master
+
+      - name: Install dependencies
+        shell: bash
+        run: |
+          curl https://nginx.org/download/nginx-1.21.3.tar.gz | gunzip | tar -x
```

Comment:
Again, just a suggestion, but here I would normally use:
-  `--fail` or `-f` (to guarantee that curl fails unless the call is successful, otherwise say a 500 lets curl continue merrily as if all was OK.  Here you immediately pipe into gunzip and tar so an invalid response would _probably_ fail immediately anyway.)
- `--location` or `-L` to follow symlinks; the server maintainer probably considers moving the data but setting up a redirect as a non-breaking change.
- `--retry 5` as network problems will otherwise occasionally kill your CI jobs.

Feel free to use any of these you feel might be appropriate and to discard the rest.

Reply:
Yep, will do. The current command is just the same as it was before, so I didn't change that. But I agree these flags make sense. Updated the code.